axelar-chains-config/info/testnet.json

```diff
@@ -2064,6 +2064,29 @@
       },
       "finality": "finalized",
       "approxFinalityWaitTime": 60
+    },
+    "linea-sepolia": {
+      "name": "Linea-Sepolia",
+      "id": "linea-sepolia",
+      "axelarId": "linea-sepolia",
+      "chainId": 59141,
+      "rpc": "https://rpc.sepolia.linea.build",
+      "tokenSymbol": "ETH",
+      "contracts": {
+        "AxelarGateway": {}
+      },
+      "explorer": {
+        "name": "Lineascan",
+        "url": "https://sepolia.lineascan.build/",
+        "api": "https://api-sepolia.lineascan.build/api"
+      },
+      "gasOptions": {
+        "gasPrice": 1000000000,
+        "gasLimit": 61000000
+      },
+      "confirmations": 4,
```

Comment:
why 4 vs default of 2 for L2s? it seems that blocks are only created if there's a tx as per the explorer, so waiting on more than 1 confirmations might not work well